platform/marvell-armhf/sonic-platform-et6448m/et6448m_plt_setup.sh

```diff
@@ -52,6 +52,9 @@ main()
 {
     fw_uboot_env_cfg
     et6448m_profile
+
+    python /etc/entropy.py &
+    /bin/sh /etc/inband_mgmt.sh
```

Comment:
what is this inband mgmt port?

Reply:
ET6448M board, down't have OOB (mgmt) port. 
To access management, we have enabled in-band management feature in one of the data port.